generator/templates/structfield.gotmpl

```diff
@@ -12,6 +12,7 @@
     {{- template "structTag" dict "Tag" "json" "Schema" $prop }}
   {{- end }}
   {{- if .XMLName }} xml:"{{ .XMLName }}"{{ end }}
+  {{- if .Example }} example:"{{ quoteTagValue .Example }}"{{ end }}
```

Comment:
could this be protected by a feature flag set by the CLI? e.g. if `--struct-tag` contains "example".

Reply:
this would save you a lot of changes in the existing tests.

Reply:
yeah, we could do that, however there's a piece of template above that will conflict. That's the part that generates the bad example tag I reported in the issue
```
  {{- range $i, $structTag := $.StructTags }}
    {{- if gt $i 0 }} {{ end }}
    {{- template "structTag" dict "Tag" $structTag "Schema" $prop }}
  {{- else }}
````

Reply:
Ah, I could do your `PrintTags` solution, it would probably be cleaner than trying to do this in the template itself.